java-checks/src/test/files/checks/security/CookieShouldNotContainSensitiveDataCheck.java

```diff
@@ -0,0 +1,59 @@
+import java.net.HttpCookie;
+import javax.servlet.http.Cookie;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpServletRequest;
+import javax.ws.rs.core.NewCookie;
+import org.apache.shiro.web.servlet.SimpleCookie;
+import org.springframework.security.web.savedrequest.SavedCookie;
+
+class S2255 {
+  private static final String VALUE = "value";
+
+  void servletCookie(){
+    Cookie cookie = new Cookie("name", "value"); // Noncompliant {{If the data stored in this cookie is sensitive, it should be stored internally in the user session.}}
+//                      ^^^^^^
```

Comment:
This comment doesn't assert anything in SonarJava. If you want to assert the columns at which the issue is raised, you should look at [how it's done in another file](https://github.com/SonarSource/sonar-java/blob/master/java-checks/src/test/files/checks/spring/RequestMappingMethodPublicCheck.java#L12).